config/default/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 # Adds namespace to all resources.
-namespace: oadp-nac-system
+namespace: oadp-nac-system # TODO change
```

Comment:
what should be our default namespace? I think I heard us think about `openshift-adp`?

Reply:
also, want to find a easy way to change this from makefile

Reply:
I think for the beginning it should *not* be openshift-adp, so we don't make mistakes where we bind this controller with the OADP one. Having at first separate namespace will ensure those two are separate and can be then installed together in one.

Reply:
will add this responsibility to the oadp integration task